asset/style/main.css

```diff
@@ -446,6 +470,39 @@ textarea {
   }
 
   .section-grid {
-    grid-template-columns: auto;
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+@media only screen and (min-width: 1021px) {
```

Comment:
- [ ] Your `header` in the desktop version overlaps the text since it has no background as you can see below. Kindly give it a background so it looks cleaner

![image](https://user-images.githubusercontent.com/90556221/166111474-303e83ea-bcbe-4be9-864b-409565acb1bb.png)

- [ ] The links for the header don't seem to be centered in the desktop version. Kindly fix it, the way to do it would be to have the `menu` outside the current `nav` and inside another `nav-mobile` container. Then you can simply use `justify-content: space-between` on the `header` to center the links.